src/ant/include/ant/AntennaChecker.hh

```diff
@@ -61,6 +64,62 @@ struct PARinfo;
 struct ARinfo;
 struct AntennaModel;
 
+///////////////////////////////////////
+// namespace gtl = boost::polygon;
+struct PinType;
+struct GraphNode;
+
+struct InfoType
+{
+  double PAR;
+  double PSR;
+  double diff_PAR;
+  double diff_PSR;
+  double area;
+  double side_area;
+  double iterm_gate_area;
+  double iterm_diff_area;
+
+  double CAR;
+  double CSR;
+  double diff_CAR;
+  double diff_CSR;
+
+  std::vector<dbITerm*> iterms;
+
+  InfoType& operator+=(const InfoType& a)
+  {
+    PAR += a.PAR;
+    PSR += a.PSR;
+    diff_PAR += a.diff_PAR;
+    diff_PSR += a.diff_PSR;
+    area += a.area;
+    side_area += a.side_area;
+    return *this;
+  }
+  InfoType()
+  {
+    PAR = 0.0;
+    PSR = 0.0;
+    diff_PAR = 0.0;
+    diff_PSR = 0.0;
+
+    area = 0.0;
+    side_area = 0.0;
+    iterm_gate_area = 0.0;
+    iterm_diff_area = 0.0;
+
+    CAR = 0.0;
+    CSR = 0.0;
+    diff_CAR = 0.0;
+    diff_CSR = 0.0;
+  }
+};
+
+typedef std::unordered_map<dbTechLayer*, InfoType> LayerInfoVector;
```

Comment:
Similar question here. What is LayerInfo? Info is a bad name because it says nothing about what it represents in the code.